src/lang/en-CA.json

```diff
@@ -21,15 +21,17 @@
 	"account.leave-organisation-help": "Before joining a new organization, you must leave your current organization.",
 	"account.leave-organisation": "Leave organization",
 	"action.confirmation": "Confirmation",
-	"action.password-reset-success": "Your password has been reset sucessfully! Please check your email.",
+	"action.password-reset-success": "Your password has been reset successfully! Please check your email.",
 	"action.confirm-password": "Confirm Password",
 	"action.share": "Share",
+	"action.share-question": "Share ",
```

Comment:
So this is here because:
in some instances the word Share is a statement and in others it's a question. In Spanish questions begin with an upside down question mark so was looking for a way to to correctly display that.:
	"action.share": "Compartir",
	"action.share-question": "¿Compartir ",
	
Also, I need for there to be a space after the word 'Share' when it was used as a question.


<img width="587" alt="share_question_english" src="https://user-images.githubusercontent.com/10123216/135519835-b78a26ab-800d-4dc2-b6a4-75773ac33f80.png">
<img width="587" alt="share_question_spanish" src="https://user-images.githubusercontent.com/10123216/135519843-09a494c1-d2f8-4519-96b5-6b38ea38370a.png">

Reply:
agreed for usage in the Spanish context. though for adding a space, an entirely new element is not necessary. the space can be added in the place where the resource is being used. you can remove the space from the files.